### DISCUSSION TITLE
Shared Layout ?

### DISCUSSION THREAD

**Discussion by CappaCoppa** [a-h/templ#528] [Q&A]
Hi just looking is there a way how to implement a shared layout thru different routes ?

**Comment by a-h** [comment]
I'm sorry, I don't fully understand what you're asking. I think when you say "routes", you mean, you want to use the same header and footer for different HTTP routes?

If so, these are probably the docs you're looking for. It covers how to compose templates, and the blog example demonstrates doing this using different HTTP routes.

https://templ.guide/syntax-and-usage/template-composition
https://github.com/a-h/templ/tree/main/examples/blog
